README.md

```diff
@@ -1,67 +1,85 @@
-Our project is an online store called E-Commerce-App, which is designed for customers to make online purchases, especially for those who are looking for convenience and favor 
-online shops over traditional brick-and-mortar stores because of their comfort. It will store data such as product details and user information, so that the consumers can log in to their own accounts and have access to a variety of online purchasing features and services. It also supports the users to read descriptions, view photographs of the products, and check user reviews as they explore through comprehensive product lists. It includes products from a variety of categories, including electronics, apparel, household items, and more, which can be searched for by users. We are also considering functionalities such as creating the seller side of accounts and pages depending on the time constraints.
+    e.  User Authentication with password assistance ✅
 
-## **Project Task Requirements**:
+2.  **Standard Requirements**
 
-1.  **Minimal Requirements**
+    a.  User Account page ✅
 
-    a.  A dashboard with a search bar with product filters
+    b.  View past orders ✅
 
-    b.  View selected product details.
+    c.  Add products to wishlists ✅
 
-    c.  Add product to shopping cart.
+    d.  An option to share products with friends ⚠️
 
-    d.  Products checkout page.
+    e.  Add product ratings ✅
 
-    e.  User Authentication with password assistance.
+    f.  Email notifications on product purchase ✅
 
-2.  **Standard Requirements**
+4.  **Stretch Requirements**
+
+    a.  Show user's recently viewed products ❌
+
+    b.  Add reviews to a product/see past reviews ✅
 
-    a.  User Account page.
+    c.  An option to return an order ✅
 
-    b.  View past orders
+    d.  Functionality to create seller/admin accounts ✅
 
-    c.  Add products to wish Lists.
+    f.  Allow seller/admin to post, delete, and update products ✅
 
-    d.  An option to share products with friends.
+    
+## Tech Stack
 
-    e.  Add product ratings
 
-    f.  Email or text notifications on product purchase
+## Above and Beyond Functionality
 
-3.  **Stretch Requirements**
 
-    a.  Show user's recently viewed products.
+## Next Steps
+- We have successfully fulfilled nearly all project requirements, with the exception of the 'Show Recently Viewed Products' feature, which remains pending, and the partially completed 'Share Products with Friends' functionality. Completing these two goals would further enhance the app's overall functionality.
+- During our discussions, we explored additional potential features to enrich the shopping site's capabilities. These exciting prospects include multi-language support for broader accessibility, implementing an advanced product recommendation algorithm personalized to each user, as well as introducing user roles to create separate pages exclusively accessible to sellers and buyers, adding depth and sophistication to our platform.
```

Comment:
Can we please have bullet points instead?

Reply:
Do you mean bullet points for each additional feature?

Reply:
Yes

Reply:
Just updated it, let me know how that looks

Reply:
It looks good, Thanks:)